### PR TITLE
Improve quality of *.ts files

### PR DESCRIPTION
**Pull request by mattpap**
- [x] enable eslint
- [x] use stricter tsconfig

### CODE REVIEW

**Comment by mattpap** [comment]
@bryevdv, I'm not sure if you get notifications for this repository. I'm going to merge this soon. Note that though tests all "all green", they actually don't tests much besides if things got built and the extension is loadable by jupyter-lab at all. I have been working on improving testing, but that will happen later on in another PR.

**Comment by bryevdv** [comment]
I should get motifs now. I will look at this tomorrow. Do you need any help with releasing/publishing?

File: src/renderer.ts [hunk 1]
```diff
@@ -1,118 +1,98 @@
...
           if (status == "restarting" || status === "dead") {
-            delete (window as any).Bokeh.embed.kernels[this._document_id];
+            delete Bokeh.embed.kernels[this._document_id!]
           }
-        }, this);
+        }, this)
       }
     } else if (metadata.server_id !== undefined) {
-      // I'm a server document
+      // I"m a server document
```
Comment (bryevdv) [inline src/renderer.ts]: `I'm`

**Comment by bryevdv** [comment]
LGTM

**Comment by mattpap** [comment]
> Do you need any help with releasing/publishing?

Not yet, I have bunch of other changes to get merged first.

**Pull request merged**